test-data/unit/check-functions.test

```diff
@@ -363,7 +363,7 @@ y = x  # E: Incompatible types in assignment (expression has type Callable[...,
 
 a, b = None, None # type: (A, B)
 a = f()     # E: Incompatible types in assignment (expression has type "B", variable has type "A")
-b = f(b)    # E: Argument 1 to "f" has incompatible type "B"; expected "A"
+b = f(b)    # E: Argument 1 to "f" has incompatible type "B"; expected "Optional[A]"
```

Comment:
Hm, I think that's just how Jukka used to write tests, with the classes/functions at the end. A lot of them have that. Anyway I'll merge now!